### PR TITLE
[Refactor:System] setup sample courses zip->shutil

### PR DESCRIPTION
**Pull request by Yash1256**
Completing the #TODO in file setup_sample_courses.py at line 42
Replaced zip with shutil.

### CODE REVIEW

**Comment by Yash1256** [comment]
@bmcutler Can you guide me why the test is failing?

**Comment by Yash1256** [comment]
@elihschiff Thanks Sir Is there any changes done from my side?

**Comment by elihschiff** [comment]
There are no changes you have to do regarding the tests, they should finish soon and this PR will be automatically marked as passing.
You can however make changes to your code to address the issues I laid out in my comment above.

**Comment by Yash1256** [comment]
@elihschiff I have edited the file with desired changes as mentioned above

File: .setup/bin/setup_sample_courses.py [hunk 1]
```diff
@@ -508,8 +505,6 @@ def create_gradeable_submission(src, dst):
     a .zip), then we will extract the contents of the source (using ZipFile) to the destination, else we
```
Comment (elihschiff) [inline .setup/bin/setup_sample_courses.py]: I was actually talking about fixing this line as it says "using ZipFile" which after your pr will no longer be used. When you removed the 3 lines at the top it switched your line count so when I said line 508 I was talking about before you removed the lines not after.

```suggestion
    a .zip), then we will extract the contents of the source (using shutil) to the destination, else we
```

File: .setup/bin/setup_sample_courses.py [hunk 2]
```diff
@@ -508,8 +505,6 @@ def create_gradeable_submission(src, dst):
...
     just do a simple copy operation of the source file to the destination location.
 
-    At this point, if we created a zip file (as part of that first step), we remove it from the /tmp directory.
```
Comment (elihschiff) [inline .setup/bin/setup_sample_courses.py]: Please undo this change as this comment is still true even after your edits to the code.

File: .setup/bin/setup_sample_courses.py (hunk 1 above)
Comment (Yash1256) [reply .setup/bin/setup_sample_courses.py]: Did that. Thanks for pointing out

File: .setup/bin/setup_sample_courses.py (hunk 2 above)
Comment (Yash1256) [reply .setup/bin/setup_sample_courses.py]: Did the change in 94841d9

**Comment by Yash1256** [comment]
@elihschiff I am looking forward to contribute to your Organisation are their any good first issues which make me familiar to the codebase

**Comment by elihschiff** [comment]
@Yash1256 anything here: https://github.com/Submitty/Submitty/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

**Comment by Yash1256** [comment]
@elihschiff Sir most of the issues are already been assigned or self-assigned and those are not yet assigned contain a #Low-priority tag in them

**Comment by elihschiff** [comment]
It is ok if they say low priority, you can still solve them
